fahclient/values.yaml

```diff
@@ -67,6 +67,10 @@ securityContext:
   runAsUser: 9999
   runAsGroup: 9999
 
+service:
+  type: ClusterIP
+  port: 80
+
```

Comment:
git commit error?

Reply:
Nope, the test fails without this. It was either add it back or remove the test.